packages/trees/backend/sirius-components-trees/src/main/java/org/eclipse/sirius/components/trees/description/TreeDescription.java

```diff
@@ -131,8 +133,8 @@ public Function<VariableManager, Object> getParentObjectProvider() {
         return this.parentObjectProvider;
     }
 
-    public Function<VariableManager, List<String>> getIconURLProvider() {
-        return this.iconURLProvider;
+    public Function<VariableManager, List<String>> getTreeItemIconProvider() {
```

Comment:
Is that identified as an API break? I would still keep the URL in there to be a bit more specific.